src/uct/api/v2/uct_v2.h

```diff
@@ -198,6 +209,49 @@ typedef struct uct_md_mem_dereg_params {
 } uct_md_mem_dereg_params_t;
 
 
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief Operation parameters passed to @ref uct_iface_is_reachable_v2.
+ */
+typedef struct uct_iface_is_reachable_params {
+    /**
+     * Mask of valid fields in this structure and operation flags, using
+     * bits from @ref uct_iface_is_reachable_v2. Fields not specified
+     * in this mask will be ignored. Provides ABI compatibility with respect
+     * to adding new fields.
+     */
+    uint64_t                     field_mask;
+
+    /**
+     * Device address to check reachability to. It is not set
+     * if iface_attr.dev_addr_len == 0, and must be set otherwise.
```

Comment:
as we discussed user may omit iface or device address. Let's soften the description like:
`Device address to check reachability to. This field is not relevant if iface_attr.dev_addr_len == 0.`
The same is relevant for iface address description

Reply:
fixed